sktime/datasets/_fpp3_loaders.py

```diff
@@ -0,0 +1,274 @@
+
+    if dataset_name == "global_economy":
+        obj.reset_index(inplace=True)
+        obj.columns = ['Year', 'Country', 'Code', 'GDP', 'Growth', 'CPI', 'Imports', \
+                       'Exports', 'Population']
+        obj.set_index(['Country', 'Year'], inplace=True)
+
+    if dataset_name == "hh_budget":
+        obj.reset_index(inplace=True)
+        obj.columns = ['Year', 'Country', 'Debt', 'DI', 'Expenditure', 'Savings', \
+                       'Wealth', 'Unemployment']
+        obj.set_index(['Country', 'Year'], inplace=True)
+
+    if dataset_name == "nyc_bikes":
+        obj['start_time'] = pd.to_datetime(obj['start_time'], unit='s')
+        obj.set_index('start_time', inplace=True)
+        obj.reset_index(inplace=True)
+        obj.set_index(['bike_id', 'start_time'], inplace=True)
+
+    if dataset_name == "pedestrian":
+        obj['Date_Time'] = pd.to_datetime(obj['Date_Time'], unit='s')
+        obj.set_index('Date_Time', inplace=True)
+        obj.reset_index(inplace=True)
+        obj['Date'] = pd.to_datetime(obj['Date'])
+        obj.set_index(['Sensor', 'Date_Time'], inplace=True)
+
+    if dataset_name == "us_employment":
+        obj.reset_index(inplace=True)
+        obj.set_index(['Series_ID', 'Month'], inplace=True)
+
+    if dataset_name == "aus_retail":
+        obj.reset_index(inplace=True)
+        obj.set_index(['State', 'Industry', 'Month'], inplace=True)
+
+    if dataset_name == "PBS":
+        obj.reset_index(inplace=True)
+        obj.set_index(['Concession', 'Type', 'ATC1', 'ATC2', 'Month'], inplace=True)
+
+    return (True, obj)
+
+def _process_dataset(dataset_name, temp_folder="/tmp"):
+    known, url = _get_dataset_url(dataset_name)
+    if known:
+        temp_dir = _decompress_file_to_temp(url=url, temp_folder=temp_folder)
+        found, path = _find_dataset(temp_dir, dataset_name)
+        if found:
+            ret, obj = _import_rda(path)
+        else:
+            return (False, None)
+
+        shutil.rmtree(temp_dir) ## cleanup
+
+        if not ret:
+            return (False, None)
+
+        do_mtype = True
+        if do_mtype:
+            result = _dataset_to_mtype(dataset_name, obj)
+            return result
+        else:
+            return (True, obj)
+    else:
+        return (False, None)
+
+
+def load_fpp3(dataset, temp_folder="/tmp"):
+    """Load a dataset from the fpp3 package.
+
+    Parameters
+    ----------
+    dataset : str - The name of the dataset to load
+    temp_folder: str - name of folder where temporary folder can be created
+         and ultimately deleted before this function returns
+    
+    Returns a 2-tuple
+    -----------------
+    status: - logical - True on success, False on failure
+    y : if status is True, then a pd.DataFrame, else None
+    """
+    from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+    dependencies_met = _check_soft_dependencies(["requests","rdata"])
```

Comment:
this line will raise an exception if the dependencies are not present. If you want to silence the exception, you have to add `severity="none"`. I would argue, the exception is more informative here, although you may want to write a clearer message to the user.